src/middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { NextResponse, NextRequest } from 'next/server'
```

Comment:
Have you verified that this middleware is actually needed? The way I see it, the fix in the context should be enough…

Reply:
Yes thank you. I was too fast when i created the PR and forgot to remove the cookie code i tried with first